modules/graphman.js

```diff
@@ -14,6 +15,7 @@ const https = require("https");
 module.exports = {
     loadedConfig: null,
     metadata: null,
+    policySchema: null,
```

Comment:
instead of accessing the _policySchema_, introduce a getter method to access it.

Reply:
done